Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -59,6 +59,7 @@
 	private FrameLayout mFrameEmulation;
 	private LinearLayout mMenuLayout;
 
+	private enum MenuType {SAVE, LOAD}
```

Comment:
Was this formatted with our coding style found [here](https://github.com/dolphin-emu/dolphin/blob/master/Source/Android/code-style-java.jar)?

As far as I'm aware, this would format as either

```java
private enum MenuType
{
  SAVE,
  LOAD,
}
```

or

```java
private enum MenuType
{
  SAVE, LOAD
}
```

Reply:
Nope. I've fixed it (and moved the enum def to below the members vars).